lib/cmds/template_generator.js

```diff
@@ -31,6 +47,47 @@ class TemplateGenerator {
       console.log(__('For more info go to http://embark.status.im').green);
     }
   }
+
+  installTemplate(templatePath, name, installPackages) {
+    utils.cd(templatePath);
+    utils.sed('package.json', '%APP_NAME%', name);
+
+    if (installPackages) {
+      console.log(__('Installing packages...').green);
+      utils.runCmd('npm install');
+    }
+
+    console.log(__('Init complete').green);
+    console.log('\n' + __('App ready at ').green + templatePath);
+  }
+
+  getExternalProject(uri) {
```

Comment:
I'm having issues understanding what all of those do. The /archive/ is from github right? So we are only gonna make available templates on github? I would have thought that for example, if it a normal link (without github in the url), we straight up download that or if .zip is already in the url.

Reply:
Also, after checking your tests, looks like you don't support branches/tags, maybe should be a good add.

Reply:
only github, at least for now. will look into the branches + tags